src/components/grapher/GrapherDots.vue

```diff
@@ -64,10 +114,21 @@ export default Vue.extend({
 </script>
 
 <style scoped lang="scss">
+.grapher-dots--dot-selected {
+  fill: $yellow;
+}
+
 .grapher-dots--dottext {
   fill: $black;
   font-size: 2px;
   text-anchor: left;
   user-select: none;
 }
+
+.grapher-dots--dottext-selected {
+  fill: $yellow;
+  font-size: 2px;
```

Comment:
nit: Can we make the font size a little smaller? It looks a little big and difficult to read when dots are 2 steps apart like this:

![image](https://user-images.githubusercontent.com/13753033/99926527-e1fd8f80-2cf6-11eb-9b6f-2dee69f9730a.png)

Reply:
Will do.